snap/snapcraft.yaml

```diff
@@ -0,0 +1,26 @@
+name: minio
+version: git
+summary: open source object storage server compatible with Amazon S3 APIs
+description: |
+  Minio is an object storage server released under Apache License v2.0. It is
+  compatible with Amazon S3 cloud storage service. It is best suited for
+  storing unstructured data such as photos, videos, log files, backups and
+  container / VM images. Size of an object can range from a few KBs to a
+  maximum of 5TB.
+
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: strict
+
+apps:
+  minio:
+    command: minio
+    plugs: [home, removable-media, network, network-bind]
+
+parts:
+  minio:
+    source: .
+    plugin: go
+    go-importpath: github.com/minio/minio
+    after: [go]
+  go:
+    source-tag: go1.7.5
```

Comment:
We have moved to go1.8.3 now you can change this back to previous version sorry @elopio